src/ui/pages/Address/index.tsx

```diff
@@ -15,56 +17,110 @@ const useStyles = makeStyles({
     margin: 30,
     boxSizing: 'border-box'
   },
-  button: {
-
+  button: {},
+  dialogTitle: {
+    textAlign: 'right'
   },
-  textField: {
-
+  dialogContent: {
+    padding: '0 24px 24px',
+    textAlign: 'center',
+    minWidth: 200
+  },
+  address: {
+    marginTop: 16
+  },
+  loading: {
+    width: 200,
+    padding: 24,
+    textAlign: 'center'
+  },
+  tip: {
+    marginBottom: 24,
+    color: 'green'
   }
 });
 
+interface AppProps {}
 
-interface AppProps { }
-
-interface AppState { }
+interface AppState {}
 
-export default function (props: AppProps, state: AppState) {
+export default function(props: AppProps, state: AppState) {
   const classes = useStyles();
   const [loading, setLoading] = React.useState(true);
   const [address, setAddress] = React.useState({});
   const [balance, setBalance] = React.useState('0');
-  const { network } = React.useContext(AppContext)
+  const [open, setOpen] = React.useState(false);
+  const [tooltip, setTooltip] = React.useState('');
+  const { network } = React.useContext(AppContext);
 
   React.useEffect(() => {
-    chrome.runtime.onMessage.addListener(function (message, sender, sendResponse) {
-      if (message.messageType === MESSAGE_TYPE.ADDRESS_INFO && message.address) {
+    chrome.runtime.onMessage.addListener(function(
+      message,
+      sender,
+      sendResponse
+    ) {
+      if (
+        message.messageType === MESSAGE_TYPE.ADDRESS_INFO &&
+        message.address
+      ) {
         setAddress(message.address);
-      // get balance by address
+        // get balance by address
       } else if (message.messageType === MESSAGE_TYPE.BALANCE_BY_ADDRESS) {
         setBalance(message.balance);
         setLoading(false);
       }
-    })
-    chrome.runtime.sendMessage({ messageType: MESSAGE_TYPE.REQUEST_ADDRESS_INFO })
+    });
+
+    chrome.runtime.sendMessage({
+      messageType: MESSAGE_TYPE.REQUEST_ADDRESS_INFO
+    });
+
     chrome.runtime.sendMessage({
       messageType: MESSAGE_TYPE.REQUEST_BALANCE_BY_ADDRESS,
       network
-    })
+    });
     setLoading(true);
-  }, [])
-  const balanceNode = loading ? <div data-testid="balance">loading</div> : <div className="balance" data-testid="balance">{balance}<span className="">CKB</span></div>
+  }, []);
 
   const history = useHistory();
   const onSendtx = () => {
-    history.push('/send-tx')
-  }
+    history.push('/send-tx');
+  };
+
+  React.useEffect(() => {
+    (async function copyAddress() {
+      if (open && address[network]) {
+        await navigator.clipboard.writeText(address[network]);
+      }
+    })();
+    setTooltip('Address has been copied to clipboard');
```

Comment:
```suggestion
    setTooltip('Address has been copied to clipboard');
```
when address is not generated，can show other hint

Reply:
I will try to resolve this problem.